docs/current/cookbook/snippets/cache-dependencies/main.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+import anyio
+
+import dagger
+
+
+async def main():
+    config = dagger.Config(log_output=sys.stdout)
```

Comment:
`stderr` please, however this whole config doesn't matter when running with `dagger run`.

Reply:
Done